96769015_Vikram/Adding.cu

```diff
@@ -0,0 +1,65 @@
+#include<iostream>
+using namespace std;
+
+__global__ void Add(float *array1, float *array2, float *out,unsigned int *i)
```

Comment:
The variable 'i' is not required since you can span the entire array in one kernel call. There is no restriction on the number of blocks that can be launched.